vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -379,18 +379,7 @@ void Contribution::CreateNewEntry(
   if (wallet_balance < queue->amount) {
     contribution->amount = wallet_balance;
     queue->amount = queue->amount - wallet_balance;
-
-    if (queue->type == ledger::RewardsType::RECURRING_TIP ||
-        queue->type == ledger::RewardsType::ONE_TIME_TIP) {
-      AdjustPublisherListAmounts(
-          std::move(queue_publishers),
-          &publishers_new,
-          &publishers_left,
-          wallet_balance);
-      queue->publishers = std::move(publishers_left);
-    } else {
-      publishers_new = std::move(queue_publishers);
-    }
+    publishers_new = std::move(queue_publishers);
```

Comment:
we don't need to adjust amount anymore as tips are always processed as one publisher and AC has calculation in other place

Reply:
Since we're setting `publishers_new` to the same thing in the `if` and `else`, maybe it can be set once outside of the `if` statement?